diff --git a/app/helpers/journals_helper.rb b/app/helpers/journals_helper.rb
index b8e7f8b..aaac353 100644
--- a/app/helpers/journals_helper.rb
+++ b/app/helpers/journals_helper.rb
@@ -32,6 +32,8 @@
                                              { :controller => 'journals', :action => 'edit', :id => journal, :format => 'js' },
                                                 :title => l(:button_edit)) if editable
     end
+    # hook for redmine_pirate_helpdesk
+    call_hook(:helper_journals_links, {:links => links, :journal => journal, :issue => issue, :project => issue.project})
     content << content_tag('div', links.join(' ').html_safe, :class => 'contextual') unless links.empty?
     content << textilizable(journal, :notes)
     css_classes = "wiki"
