diff --git a/app/models/mailer.rb b/app/models/mailer.rb
index 7166025..9fc2f5e 100644
--- a/app/models/mailer.rb
+++ b/app/models/mailer.rb
@@ -33,19 +33,30 @@ class Mailer < ActionMailer::Base
   #   issue_add(issue) => Mail::Message object
   #   Mailer.issue_add(issue).deliver => sends an email to issue recipients
   def issue_add(issue)
+    # XXX: dirty hack for mailinglist notifications, please improve this!
+    no_mailinglist = true
+    if issue.status.name == "Bestätigt"
+      no_mailinglist = nil
+    end
+    # end hack
     redmine_headers 'Project' => issue.project.identifier,
                     'Issue-Id' => issue.id,
                     'Issue-Author' => issue.author.login
     redmine_headers 'Issue-Assignee' => issue.assigned_to.login if issue.assigned_to
+    # XXX: dirty hack for mailinglist notifications, please improve this!
+    redmine_headers 'No-Mailinglist' => no_mailinglist
+    Rails.logger.info("### no_mailinglist #{no_mailinglist}")
+    # end hack
     message_id issue
     @author = issue.author
     @issue = issue
     @issue_url = url_for(:controller => 'issues', :action => 'show', :id => issue)
     recipients = issue.recipients
     cc = issue.watcher_recipients - recipients
+    subject = ((issue.project.ancestors + [issue.project]).collect { |p| p.name }).join("/")
     mail :to => recipients,
       :cc => cc,
-      :subject => "[#{issue.project.name} - #{issue.tracker.name} ##{issue.id}] (#{issue.status.name}) #{issue.subject}"
+      :subject => "[#{subject} - #{issue.tracker.name} ##{issue.id}] (#{issue.status.name}) #{issue.subject}"
   end
 
   # Builds a Mail::Message object used to email recipients of the edited issue.
@@ -55,10 +66,36 @@ class Mailer < ActionMailer::Base
   #   Mailer.issue_edit(journal).deliver => sends an email to issue recipients
   def issue_edit(journal)
     issue = journal.journalized.reload
+    # XXX: dirty hack for mailinglist notifications, please improve this!
+    no_mailinglist = true
+    if issue.status.name == "Bestätigt"
+        if (journal.new_value_for('status_id') \
+            or journal.new_value_for('description') \
+            or journal.new_value_for('Veranstaltungsort') \
+            or journal.new_value_for('Ende') \
+            or journal.new_value_for('Adresse') \
+            or journal.new_value_for('Veranstaltungsort') \
+            or journal.new_value_for('Startzeit') \
+            or journal.new_value_for('start_date') \
+            or journal.new_value_for('due_date'))
+
+          no_mailinglist = nil
+        end
+    elsif issue.status.name == "Abgesagt"
+        if journal.new_value_for('status_id') 
+          no_mailinglist = nil
+        end
+    end
+    # end hack
     redmine_headers 'Project' => issue.project.identifier,
                     'Issue-Id' => issue.id,
                     'Issue-Author' => issue.author.login
     redmine_headers 'Issue-Assignee' => issue.assigned_to.login if issue.assigned_to
+    # XXX: dirty hack for mailinglist notifications, please improve this!
+    redmine_headers 'No-Mailinglist' => no_mailinglist
+    redmine_headers 'Issue-Status' => issue.status.name
+    Rails.logger.info("no mailinglist header? '#{no_mailinglist}' #{issue.status.name}")
+    # end hack
     message_id journal
     references issue
     @author = journal.user
