diff --git a/app/models/mail_handler.rb b/app/models/mail_handler.rb
index 8421fb6..d665454 100644
--- a/app/models/mail_handler.rb
+++ b/app/models/mail_handler.rb
@@ -177,9 +177,21 @@ class MailHandler < ActionMailer::Base
       issue.subject = '(no subject)'
     end
     issue.description = cleaned_up_text_body
-
+    
     # add To and Cc as watchers before saving so the watchers can reply to Redmine
     add_watchers(issue)
+    ### patch for redmine_pirate_helpdesk to add sender mail to created tickets
+    ### TODO: move to plugin
+    custom_field = CustomField.find_by_name('Absender-Mail')
+    custom_value = CustomValue.find(
+      :first,
+      :conditions => ["customized_id = ? AND custom_field_id = ?", issue.id, custom_field.id]
+    )
+    if custom_value
+      custom_value.value = issue.author.mail
+    end
+    ### end patch
+    custom_value.save()
     issue.save!
     add_attachments(issue)
     logger.info "MailHandler: issue ##{issue.id} created by #{user}" if logger && logger.info
